gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Driver/GremlinClientTests.cs

```diff
@@ -285,12 +285,22 @@ public async Task ShouldSaveVariableBetweenRequestsInSession()
             var gremlinServer = new GremlinServer(TestHost, TestPort);
             using (var gremlinClient = new GremlinClient(gremlinServer, sessionId: Guid.NewGuid().ToString()))
             {
-
                 await gremlinClient.SubmitWithSingleResultAsync<int>("x = 1");
 
                 var response = await gremlinClient.SubmitWithSingleResultAsync<int>("x + 2");
                 Assert.Equal(3, response);
             }
         }
+
+        [Fact]
+        public void ShouldThrowOnExecutionOfMultiConnectionInPool()
```

Comment:
The test looks good in general, but it is not really an integration test as it works without any external dependencies like a Gremlin Server instance. Could you therefore please move it into the `Gremlin.Net.UnitTest` project? You can create a new class `GremlinClientTests` in the `Driver` directory there as there probably are no unit tests yet for this class.

Reply:
ok, it's UnitTest, thanks